aws/resource_aws_rds_cluster.go

```diff
@@ -222,6 +224,29 @@ func resourceAwsRDSCluster() *schema.Resource {
 				Optional: true,
 			},
 
+			"iam_role": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"role_arn": {
+							Type:     schema.TypeString,
+							Required: true,
+						},
+						"role_status": {
+							Type:     schema.TypeString,
+							Computed: true,
```

Comment:
Computed values in sets are a bit weird, since it's pretty hard to actually use them (no first-class interpolation syntax for set members) and it makes diffs confusing.

If interpolating the role status is important I'd suggest instead having a separate attribute `iam_role_statuses` that is a computed map of strings, and expose them in there with the role arn as the key. This way the syntax for interpolating them will be more convenient, and perhaps also the `iam_roles` attribute itself could be just a set of strings.